pvr.nextpvr/changelog.txt

```diff
@@ -1,3 +1,6 @@
+v21.0.0
+- Connection change, start in conneecting state, defer unreaachble.
```

Comment:
Spelling mistake

Reply:
Two mistakes, I guess I shouldn't have submitted this with 2 hours sleep coming home from vacation, but I was keen to try the change.  Thanks a lot.